scale-tests/deploy-dispatchers.py

```diff
@@ -116,11 +115,13 @@ def deploy_dispatchers(
 
             # create dispatcher & driver role quotas
             dispatcher_role = "{}-dispatcher-role".format(service_name)
-            create_quota(name=dispatcher_role,
-                         cpus=quota_dispatcher_cpus, gpus=quota_dispatcher_gpus, mem=quota_dispatcher_mem)
+            if create_quotas:
+              create_quota(name=dispatcher_role,
+                           cpus=quota_dispatcher_cpus, gpus=quota_dispatcher_gpus, mem=quota_dispatcher_mem)
             driver_role = "{}-driver-role".format(service_name)
-            create_quota(name=driver_role,
-                         cpus=quota_driver_cpus, gpus=quota_driver_gpus, mem=quota_driver_mem)
+            if create_quotas:
```

Comment:
no need to check twice, `if create_quotas` is true on L118 you can create quota for both dispatchers and driver